devops/templates/create-env-step-template.yml

```diff
@@ -43,7 +43,14 @@ steps:
   - bash: |
       source activate ${{parameters.condaEnv}} 
       pip install --upgrade pip
-    displayName: Upgrade pip to latest
+    displayName: Upgrade pip to latest for linux or macos
+    condition:  or(eq(variables['Agent.OS'], 'Linux'), eq(variables['Agent.OS'], 'Darwin'))
+
+  - bash: |
+      source activate ${{parameters.condaEnv}} 
```

Comment:
getting random permissions errors on windows build, searched the web and using --user was recommended